app/assets/stylesheets/components/_photo-grid.scss

```diff
@@ -0,0 +1,34 @@
+.row{
```

Comment:
Be careful here. You don't want to over-ride bootstrap classes

Reply:
good call, I fixed the class names so they won't mess with bootystrap